examples/tutorials/translations/german/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>Schauen wir die Metadaten an!&nbsp;</p><p>Es gibt zwei Hauptattribute, die für <em>pointer</em> spezifisch sind:&nbsp;</p><ul><li><code>x_ptr.location : bob</code>, der Ort, also ein Verweis auf den Ort, auf den der Zeiger zeigt&nbsp;</li><li><code>x_ptr.id_at_location : &lt;random integer&gt;</code>, die ID des Tensors an diesem Ort </li></ul><p>Sie werden im Format "@" gedruckt&nbsp;</p><p>Es gibt auch andere allgemeinere Merkmale:&nbsp;</p><ul><li><code>x_ptr.id : &lt;random integer&gt;</code>, die ID unseres <em>pointer </em>Tensors, zufällig zugewiesen&nbsp;</li><li><code>x_ptr.owner : "me"</code>, der Arbeiter, dem der Zeigertensor gehört, in diesem Fall ist es der lokale Arbeiter mit dem Namen "ich"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>